torch/nn/modules/conv.py

```diff
@@ -682,8 +685,10 @@ def forward(self, input: Tensor, output_size: Optional[List[int]] = None) -> Ten
         if self.padding_mode != 'zeros':
             raise ValueError('Only `zeros` padding mode is supported for ConvTranspose1d')
 
+        # One cannot replace List by Sequence in "_output_padding" because TorchScript does
+        # not support Sequence[T]. Therefore, a "type:ignore[...]" is required.
```

Comment:
yeah. since Tuple doesn't work either. for now let's change the comment to:
```suggestion
        # One cannot replace List by Tuple or Sequence in "_output_padding" because 
        # TorchScript does not support `Sequence[T]` or `Tuple[T, ...]`.
```
please also change the other comments as well. thanks :-)